specification/nhais-adaptor.yaml

```diff
@@ -170,20 +196,26 @@ paths:
 
         https://digital.nhs.uk/services/nhais/guide-to-nhais-gp-links-documentation
 
-        Three fields are deprecated and not supported by the adaptor:
+        Three fields are DEPRECATED and not supported by the adaptor:
 
         * RPP Mileage
         * Blocked Route/Special District Marker
         * Walking Units
 
+        REQUIRED - If any of these data items are not provided or are blank then the adaptor will produce a 400 error
+        OPTIONAL - These data items may be included if available
+        DEPRECATED - The adaptor provides no FHIR representation for these items and it is not possible to include them
+        NOT REQUIRED - These data items are will be ignored if provided
 
+        Any other values not listed in the tables below and provided in the request that are valid for a FHIR R4
+        Parameter or Patient resource will be ignored.
```

Comment:
GP Suppliers should consider adding their own validation around OPTIONAL, DEPRECATED and NOT REQUIRED field types as NO VALIDATION IS APPLIED BY THE ADAPTOR.

Reply:
as discussed in a separate PR we'll add a section to the top-level description about input validation that covers all transaction types